modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/utils/lang.es.js

```diff
@@ -46,4 +46,14 @@ export const sub = (langKey, args) => {
 	return keyArray.join('');
 };
 
+export const getPluralMessage = (
+	singular,
+	plural,
+	count = 0,
+	toString,
+	subArray
+) => {
+	const message = count === 1 ? singular : plural;
```

Comment:
@interaminense , this verification may be wrong, if the count is equal to 0.

Example;

Count = 0 -> x-fields
Count = 1 -> x-field
Count > 1 -> x-fields

This condition should be count <= 1 ? singular : plural

Reply:
Yes, good observation, Keven! I'll update it to `count > 1 ? plural : singular`

Reply:
Sure, much better 👍🏻